### ISSUE TITLE
Convert-AlcOutputToAzureDevOps: wrong error output

### ISSUE THREAD

**Issue by IceOnly** [microsoft/navcontainerhelper#1542]
**Describe the issue**
Convert-AlcOutputToAzureDevOps shows no file path, line no. and position of an error output.

**Sample**
```
##[error]An application object of type 'Report' with ID '50000' is already declared by the extension 'Test_App (1.0.28.1)'
##[error]An application object of type 'Report' with ID '50000' is already declared by the extension 'Test_App (1.0.28.1)'
```

Should look like:
```
##vso[task.logissue type=error;sourcepath=C:\Repos\Test_App\Reports\A.Report.al;linenumber=1;columnnumber=8;code=AL0264;]An application object of type 'Report' with ID '50000' is already declared by the extension 'Test_App (1.0.28.1)'
##vso[task.logissue type=error;sourcepath=C:\Repos\Test_App\Reports\B.Report.al;linenumber=1;columnnumber=8;code=AL0264;]An application object of type 'Report' with ID '50000' is already declared by the extension 'Test_App (1.0.28.1)'
```

**Comment by freddydk** [comment]
To repro this, you have one app with two reports using the same ID?

**Comment by IceOnly** [comment]
For this speficific case, yes. But every error will have no information about the file that has causing it.
The both case statement matches an error. without breaks the last on will win. And in this case it is the one without file informations .

**Comment by freddydk** [comment]
Looking at your PR, it looks correct and without any side effects.

**Comment by freddydk** [comment]
Shipped in BcContainerHelper 2.0.0
